flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/NettyShuffleEnvironmentConfiguration.java

```diff
@@ -158,6 +166,8 @@ public static NettyShuffleEnvironmentConfiguration fromConfiguration(
 
 		boolean isNetworkDetailedMetrics = configuration.getBoolean(NettyShuffleEnvironmentOptions.NETWORK_DETAILED_METRICS);
 
+		boolean isReleasingExternallyManagedPartitionsOnConsumptions = configuration.getBoolean(NettyShuffleEnvironmentOptions.RELEASE_EXTERNALLY_MANAGED_PARTITIONS_ON_CONSUMPTION);
```

Comment:
This line is also too long and may be split into two lines

Reply:
addressed in #8654.